app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/savedReplySelector.tsx

```
@@ -0,0 +1,112 @@
+import { MessageSquareText as SavedReplyIcon, ChevronDown, Search } from "lucide-react";
+import { useState, useRef, useEffect } from "react";
+import { Button } from "@/components/ui/button";
+import {
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
```

ESLint

[error] 'CommandInput' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)